web-client/src/presenter/presenter.ts

```diff
@@ -1153,8 +1153,7 @@ export const presenterSequences = {
   setCurrentPageIndexSequence:
     setCurrentPageIndexSequence as unknown as Function,
   setCustomCaseReportFiltersSequence,
-  setDocumentForPreviewSequence:
-    setDocumentForPreviewSequence as unknown as Function,
+  setDocumentForPreviewSequence,
```

Comment:
Why was this type removed?

Reply:
The type was set in the sequence itself.